roles/tendrl-storage-node/tasks/rsyslog.yml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
This is defined in tedrl-storage-node and used in tendrl-server?
You need to have this at both places right?

Reply:
Yes, During the test i used. Forget to git add. I will add the file

Reply:
Done